index-sample.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sample</title>
+    <link rel="stylesheet" href="/style-sample.css">
+</head>
+<body>
+    <div class="app-sample"> 
+        <img src="/images/imgQuotes.png" alt="img" class="section-quotes">
+        <div class="section-heading">
+            <h2 class="section-title">Real Stories from Real Customers</h2>   
+            <p class="section-desp">Get inspired by these stories.</p>                 
+        </div>        
+        <div class="section-testimonial3">
+            <img src="/images/imgHubspot.png" alt="img" class="testimonial-image">
+            <div class="section-content">
+                <img src="/images/imgQuotes.png" alt="img" class="section-quotes-testimonial">
+                <div class="section-feedbackPerson">
```

Comment:
Classnames should be in kebab case unless it's rendered using CSS modules or sass.
Update all the occurrences accordingly.